integrations/google_vertex/src/haystack_integrations/components/embedders/google_vertex/document_embedder.py

```diff
@@ -205,11 +205,9 @@ def run(self, documents: List[Document]):
         """
         Processes all documents in batches while adhering to the API's token limit per request.
 
-        Args:
-        - documents: List of documents (strings) to be processed.
+        :param documents: A list of documents to embed.
 
-        Returns:
-        - List of dictionaries containing processing results for each document.
+        :returns: A list of processed documents with embeddings.
```

Comment:
```suggestion
        :returns:
            A dictionary with the following keys:
            - `documents`: A list of documents with embeddings.
```

This should be more consistent with other embedders, no?